app/controllers/ai_annotations_controller.rb

```diff
@@ -2,4 +2,19 @@ class AiAnnotationsController < ApplicationController
   def show
     @ai_annotation = AiAnnotation.find_by!(uuid: params[:uuid]).content
   end
+
+  def new; end
+
+  def create
+    text = params[:text]
+    prompt = params[:prompt]
+    inline_annotation = AiAnnotation.generate!(text, prompt)
+    ai_annotation = AiAnnotation.create!(content: inline_annotation)
```

Comment:
```suggestion
    ai_annotation = AiAnnotation.generate!(text, prompt)
```
という感じで、２つのメソッドにわけなくても良さそうに思います。
generateとcreateという似たメソッド名が並んでいると、ちょっと混乱します。

Reply:
確かに分かりづらくなってしまっていますね。
修正しました。
https://github.com/yush-nh/textae-ai/pull/12/commits/6fdc6cd6e86f464b9afba61bfc3ef0a83f1da559